ocs_ci/deployment/deployment.py

```diff
@@ -327,6 +327,41 @@ def deploy_ocs_via_operator(self):
         )
         run_cmd(f"oc create -f {cluster_data_yaml.name}", timeout=2400)
 
+    def deployment_with_ui(self):
```

Comment:
Can this work with different browsers? Firefox, Edge, Safari?....

Reply:
@shyRozen 
This should be taken care of by the openshift console part.

Reply:
we need to check if we can pass args between the two framework. Even if we can the script that gets the args (test.-gui.sh) is pretty simple and I need to check if it can be changed. I know for sure that the driver is defined in protractor.conf.ts for all of the tests in console as chrome driver. We need to understand if we can pass the args from our framework --> test-gui --> to our code to change the driver. For now I think there isn't any implementation to firefox (called gecko driver) in all the framework.
Tomorrow I'll consult with Radim

Reply:
@hnallurv currently the automation will handle just Chrome, the other browser can be added in follow up PR and we will need to have some condition to set different env vars for it and have prepared the all the requirements/deps installed in jenkins slave.

@shyRozen if you can gather all info what is needed for firefox and add it to the shared document which we have mentioned in the ticket would be great. Thanks